packages/vue-app/template/router.js

```diff
@@ -114,8 +114,12 @@ function decodeObj(obj) {
   }
 }
 
-export function createRouter () {
-  const router = new Router(routerOptions)
+export function createRouter (ssrContext, config) {
+  let base = config.router && config.router.base
```

Comment:
Not sure but don't we directly have access to `$config` from ssrContext?

Reply:
Yes, but only on server side.